docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Guide to debugging a Sample Ratio Mismatch (SRM)
+description: What to do if you have a Sample Ratio Mismatch (SRM)
+---
+
+You may be visiting this page from the [Help Center Article](https://help.amplitude.com/hc/en-us/articles/8043418569371-Sample-ratio-mismatches-Debug-issues-with-experiment-allocations) or from the [Blog Post](// TODO (AKHIL) add link here once published). A standard question you may have is what do I do that my experiment has an SRM and I understand what an SRM is.
```

Comment:
My plan was to first release this guide and then release the blog post. And then go through all the 3 docs (blog post, help center article, dev docs article) and have them all link to each other. Here is my JIRA for doing that https://amplitude.atlassian.net/browse/SKY-1860